projects/module-3/rancid-tomatillos-v3.md

```diff
@@ -232,21 +237,19 @@ It is _far_ more important to be very, very solid on the project stated learning
 Once you have decided on the area to continue learning, decide on a new feature or refactor.
 
 <section class="call-to-action">
-### DELIVERABLE  
+### WEEK 2 DELIVERABLES
 On Friday of Week 2, send a specific outline of features/work/goals to your instructors in the group DM. Please include user stories, too. Do not begin work until you get the go-ahead from an instructor. (We may help you make your goal more specific or achievable.)
 
 Here are some ideas:
 * **More React and/or Router practice:**
   - Add a search/filtering functionality for movies
 * **More testing practice:**
-  - Take time to carefully ensure that you have tested all possible views and user flows
-  - Check to see if you have a test for each user story (using the story's acceptance criteria as a guide)
-  - Supplement your Cypress tests with unit tests and integration tests by using the React Testing Library
+  - Supplement your Cypress tests with a non-trivial amount of unit tests and integration tests by using the React Testing Library 
+  - Implement 2-3 new concepts from the Cypress library that have not been covered in lessons (ie Command)
```

Comment:
Maybe we could link to the cypress docs for these other concepts?

Reply:
Love this idea. Will do!